tasks/phpstan/phpstan.neon

```diff
@@ -30,7 +30,8 @@ parameters:
     - '#Function members_register_.+ not found#'
     - '#Call to an undefined method MailPoetVendor\\Idiorm\\IdiormResultSet::set\(\)#'
     - '#Argument of an invalid type MailPoetVendor\\pQuery\\IQuery supplied for foreach, only iterables are supported#'
-    - '/(with|has) no (return )?typehint specified/' # exclude level 6 errors
+    - '/has no (return )?type specified/' # exclude level 6 errors
+    - '/has parameter (\$[_A-Z]{1}[_a-z]+)? with no type (specified)?/i' # exclude level 6 errors
```

Comment:
I'm not sure if we still want to continue ignoring type errors, but I added these for backward compatibility